pkgs/applications/terminal-emulators/iterm2/default.nix

```diff
@@ -1,48 +1,35 @@
-{ lib, stdenvNoCC, fetchFromGitHub }:
+{ lib, pkgs, stdenv }:
 
  /*
- This derivation is impure: it relies on an Xcode toolchain being installed
- and available in the expected place. The values of sandboxProfile
- are copied pretty directly from the MacVim derivation, which
- is also impure. In order to build you at least need the `sandbox`
- option set to `relaxed` or `false`.
+ This cannot be built from source as it requires entitlements and
+ for that it needs to be code signed. Automatic updates will have
+ to be disabled via preferences instead of at build time.
```

Comment:
I think disabling automatic updates makes more sense on the user/machine level using [home-manager](https://rycee.gitlab.io/home-manager/options.html#opt-targets.darwin.defaults)/nix-darwin rather than on the package level.

Reply:
> home-manager/nix-darwin

not everyone is using that and ideally updates are already disabled on the package level because they normally do not function correct anyway.

Reply:
I agree. But if you can't disable the automatic updates on the package level like in this case, it is still nice to know that you can do it with home-manager/nix-darwin as well.